txpool/txpool.go

```diff
@@ -564,6 +564,7 @@ func (p *TxPool) addTx(origin txOrigin, tx *types.Transaction) error {
 
 	// check if already known
 	if _, ok := p.index.get(tx.Hash); ok {
+		fmt.Println("known tx", "hash", tx.Hash.String())
```

Comment:
why do we need this log

Reply:
Debug print for the GH test runs (got what I needed)

[Removed](https://github.com/0xPolygon/polygon-edge/pull/513/commits/ce66280be29bcea7c9e73f297981df266ec3910b)